docs/extras/modules/data_connection/vectorstores/integrations/azuresearch.ipynb

```diff
@@ -6,14 +6,14 @@
    "source": [
     "# Azure Cognitive Search\n",
     "\n",
-    ">[Azure Cognitive Search](https://learn.microsoft.com/en-us/azure/search/search-what-is-azure-search) (formerly known as `Azure Search`) is a cloud search service that gives developers infrastructure, APIs, and tools for building a rich search experience over private, heterogeneous content in web, mobile, and enterprise applications.\n"
+    "[Azure Cognitive Search](https://learn.microsoft.com/en-us/azure/search/search-what-is-azure-search) (formerly known as `Azure Search`) is a cloud search service that gives developers infrastructure, APIs, and tools for building a rich search experience over private, heterogeneous content in web, mobile, and enterprise applications.\n"
```

Comment:
please remove the "en-us" so the link will default to the user's locale

Reply:
done, ty